src/java.base/share/classes/sun/security/pkcs/SignerInfo.java

```diff
@@ -129,6 +129,15 @@ public SignerInfo(DerInputStream derin)
     /**
      * Parses a PKCS#7 signer info.
      *
+     * SignerInfo ::= SEQUENCE {
+     *         version CMSVersion,
+     *         sid SignerIdentifier,
+     *         digestAlgorithm DigestAlgorithmIdentifier,
+     *         signedAttrs [0] IMPLICIT SignedAttributes OPTIONAL,
+     *         signatureAlgorithm SignatureAlgorithmIdentifier,
+     *         signature SignatureValue,
+     *         unsignedAttrs [1] IMPLICIT UnsignedAttributes OPTIONAL }
+     *
```

Comment:
Similar to earlier comment, existing variable naming matches better with the ASN.1 definition in RFC 2315, e.g. SignerInfo ::= SEQUENCE {
     version Version,
     issuerAndSerialNumber IssuerAndSerialNumber,
     digestAlgorithm DigestAlgorithmIdentifier,
     authenticatedAttributes
       [0] IMPLICIT Attributes OPTIONAL,
     digestEncryptionAlgorithm
       DigestEncryptionAlgorithmIdentifier,
     encryptedDigest EncryptedDigest,
     unauthenticatedAttributes
       [1] IMPLICIT Attributes OPTIONAL }

Reply:
OK.